src/components/product.js

```diff
@@ -725,11 +727,11 @@ export default class Product extends Component {
    */
   setDefaultVariant(model) {
     let selectedVariant;
+    if (this.defaultStorefrontVariantId) {
+      selectedVariant = model.variants.find((variant) => btoa(`gid://shopify/ProductVariant/${variant.id}`) === this.defaultStorefrontVariantId);
```

Comment:
maybe export `normalizeId` from `normalizeConfig` too, so you can share logic

Reply:
`normalizeId` is a little more complex than this scenario - it manages variantId when on a config object, whereas this context has no `config`.

might need to rework the normalizers to be a little more reusable.